chat-plugins/gold.js

```diff
@@ -1414,4 +1414,4 @@ function htmlfix(target) {
             target = target.substring(0, target.indexOf(fixings[u])) + '< ' + target.substring(target.indexOf(fixings[u]) + 1);
     }
     return target;
-}
\ No newline at end of file
+}
```

Comment:
This trailing whitespace was intentional for ``npm test`` to have one less thing to complain about